src/main/java/org/eclipse/basyx/aas/factory/aasx/AASXToMetamodelConverter.java

```diff
@@ -276,11 +276,32 @@ public void unzipRelatedFiles() throws IOException, ParserConfigurationException
 		List<String> files = parseReferencedFilePathsFromAASX();
 		for (String filePath : files) {
 			// name of the folder
-			unzipFile(filePath, aasxRoot);
+			unzipFile(filePath, aasxRoot, null);
```

Comment:
Wouldn't it be cleaner to just call unzipRelatedFiles(getRootFolder()) here and thus (i) completely replace the old unzipRelatedFiles and (ii) make the null check in unzipFile obsolete?